Mage.Server/src/test/java/mage/server/util/Builders.java

```diff
@@ -0,0 +1,278 @@
+package mage.server.util;
+
+import mage.server.util.config.*;
+import mage.utils.FluentBuilder;
+
+import java.math.BigInteger;
+import java.util.Collections;
+import java.util.List;
+
+public final class Builders {
```

Comment:
These are just test builders used for testing. Nothing used in the main application.